frontend/packages/integration-tests-cypress/tests/app/poll-console-updates.cy.ts

```diff
@@ -96,7 +96,7 @@ describe('PollConsoleUpdates Test', () => {
     cy.wait(`@${CHECK_UPDATES_ALIAS}`, WAIT_OPTIONS);
     cy.intercept(PLUGIN_MANIFEST_URL2, { forceNetworkError: true }).as(CHECK_MANIFEST_ALIAS2);
     cy.wait(`@${CHECK_MANIFEST_ALIAS2}`, WAIT_OPTIONS);
-    // cy.wait(`@${CHECK_MANIFEST_ALIAS2}`, WAIT_OPTIONS).should('have.property', 'error');
+    cy.wait(`@${CHECK_MANIFEST_ALIAS2}`, WAIT_OPTIONS).should('have.property', 'error');
```

Comment:
This was mistakenly commented out in https://github.com/openshift/console/pull/13070